examples/preprocessing/plot_fnirs_artifact_removal.py

```diff
@@ -75,7 +74,7 @@
 # This approach corrects baseline shift and spike artifacts without the need
 # for any user-supplied parameters :footcite:`FishburnEtAl2019`.
 
-corrected_tddr = temporal_derivative_distribution_repair(corrupted_od)
+corrected_tddr = tddr(corrupted_od)
```

Comment:
Can we leave this as it was? As a tutorial I think it is appropriate to have the extended name for ease of reading.

Reply:
Sure!

Reply:
+1 on using the long name here, and potentially adding a comment that `from ... import tddr` is also available if preferred